backend/de.metas.business/src/main/java/de/metas/order/impl/OrderLineBL.java

```diff
@@ -998,4 +1001,13 @@ private void setBPartnerLocation(@NonNull final I_C_OrderLine orderLine, @Nullab
 		OrderLineDocumentLocationAdapterFactory.locationAdapter(orderLine).setLocationAndResetRenderedAddress(bpartnerLocationAndCaptureId);
 	}
 
+	@Override
+	public boolean isOnConsignmentOrderLine(@NonNull final I_C_OrderLine orderLine)
+	{
+		final OnConsignmentAttributeRepository onConsignmentAttributeRepository = SpringContextHolder.instance.getBean(OnConsignmentAttributeRepository.class);
+
+		final I_M_AttributeSetInstance productSI = orderLine.getM_AttributeSetInstance();
+		final Boolean isOnConsignment = onConsignmentAttributeRepository.isOnConsignment(productSI);
+		return BooleanUtils.toBoolean(isOnConsignment);
```

Comment:
what is this?! `BooleanUtils.toBoolean(isOnConsignment);`

Reply:
reminiscence.. 🤦🏼